src/main/java/com/indeed/operators/rabbitmq/api/RabbitMQApiClient.java

```diff
@@ -163,8 +161,8 @@ private String buildAuthorizationHeader(final RabbitMQConnectionInfo connectionI
 
         Preconditions.checkNotNull(secret, String.format("Could not find secret with name [%s] in namespace [%s]", secretName, connectionInfo.getNamespace()));
 
-        final String username = secretsController.decodeSecretPayload(secret.getData().get("default-username"));
-        final String password = secretsController.decodeSecretPayload(secret.getData().get("default-password"));
+        final String username = secretsController.decodeSecretPayload(secret.getData().get("username"));
```

Comment:
the fact that we ever used "default-username" and "default-password" was a little silly :)